docs/howto/tracking-user-activity.rst

```diff
@@ -0,0 +1,36 @@
+..  Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+..    http://www.apache.org/licenses/LICENSE-2.0
+
+..  Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+Tracking User Activity
+=============================
+
+You can configure Airflow to route anonymous data to
+`Google Analytics <https://analytics.google.com/>`_,
+`Segment <https://segment.com/>`_, or `Metarouter <https://www.metarouter.io/>`_.
+
+Edit ``airflow.cfg`` and set the ``analytics`` block to have a ``tool`` and ``id``:
+
+.. code-block:: python
+
+  [analytics]
+  # Send anonymous user activity to Google Analytics, Segment, or Metarouter
+  # tool = google_analytics # valid options: google_analytics, segment, metarouter
```

Comment:
```suggestion
  tool = google_analytics # valid options: google_analytics, segment, metarouter
```

Upon first reading this I skimmed over this line. I think removing the leading comment would help